test/letters/containers/Main.unit.spec.jsx

```diff
@@ -1,55 +1,134 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
-import _ from 'lodash';
+import sinon from 'sinon';
 
 import { Main } from '../../../src/js/letters/containers/Main';
 
+const getBenefitSummaryOptionsSpy = sinon.spy();
+const getLetterListSpy = sinon.spy();
+const getMailingAddressSpy = sinon.spy();
+const getAddressCountriesSpy = sinon.spy();
+const getAddressStateSpy = sinon.spy();
+
 const defaultProps = {
+  letters: { },
+  destination: { },
   lettersAvailability: 'available',
-  letters: { }
+  benefitSummaryOptions: {
+    benefitInfo: '',
+    serviceInfo: '',
+  },
+  optionsAvailable: {},
+  getBenefitSummaryOptions: getBenefitSummaryOptionsSpy,
+  getLetterList: getLetterListSpy,
+  getMailingAddress: getMailingAddressSpy,
+  getAddressCountries: getAddressCountriesSpy,
+  getAddressStates: getAddressStateSpy
 };
 
+const testText = 'test child element';
+const children = (<span>{testText}</span>);
+
 describe('<Main>', () => {
-  it('should render', () => {
+  it('renders', () => {
     const tree = SkinDeep.shallowRender(<Main {...defaultProps}/>);
     const vdom = tree.getRenderOutput();
     expect(vdom).to.not.be.undefined;
   });
 
-  it('should not show loading spinner when letters are available', () => {
-    const props = _.merge({}, defaultProps, { lettersAvailability: 'available' });
+  it('renders its children when letters are available', () => {
+    const props = { ...defaultProps, children };
     const tree = SkinDeep.shallowRender(<Main {...props}/>);
-    expect(tree.subTree('LoadingIndicator')).to.be.false;
+    const childText = tree.subTree('span').text();
+    expect(childText).to.equal(testText);
   });
 
-  it('should show loading spinner when waiting for response', () => {
-    const props = _.merge({}, defaultProps, { lettersAvailability: 'awaitingResponse' });
+  it('shows a loading spinner when awaiting response', () => {
+    const props = { ...defaultProps, lettersAvailability: 'awaitingResponse' };
     const tree = SkinDeep.shallowRender(<Main {...props}/>);
     expect(tree.subTree('LoadingIndicator')).to.be.ok;
```

Comment:
I know this wasn't you, but can we change this to `expect(...).to.not.be.false`? [SkinDeep's docs](https://github.com/glenjamin/skin-deep#treesubtreeselector--matcher) say that `.subTree()` will return a tree or false. [Chai's docs](http://chaijs.com/api/bdd/#method_ok) say that the `ok` assertion isn't recommended.

Reply:
Yeah...I'm with you on this, although I'd rather do something more straightforward. Like `expect(tree.find('LoadingIndicator').length).to.equal(1)`, but...pretty sure yours is our best option.